doc/active_docs/Account Management API.json

```diff
@@ -1,8604 +1,9942 @@
-              "description": "ID of the metric.",
-              "dataType": "int",
-              "required": true,
-              "paramType": "path",
-              "threescale_name": "metric_ids"
-            },
-            {
-              "name": "id",
-              "description": "ID of the limit.",
-              "dataType": "int",
-              "required": true,
-              "paramType": "path"
-            },
-            {
-              "name": "period",
-              "description": "Period of the limit.",
-              "dataType": "string",
-              "required": true,
-              "paramType": "query",
-              "defaultValue": "minute",
-              "allowableValues": {
-                "values": [
-                  "eternity",
-                  "year",
-                  "month",
-                  "week",
-                  "day",
-                  "hour",
-                  "minute"
-                ],
-                "valueType": "LIST"
+            }
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "success",
+            "content": {}
+          }
+        }
+      },
+      "delete": {
+        "summary": "Account Delete ",
+        "tags": ["Accounts"],
+        "parameters": [
+          {
+            "name": "access_token",
+            "in": "query",
+            "description": "A personal Access Token",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "id",
+            "in": "path",
+            "description": "ID of the account.",
+            "required": true,
+            "schema": {
+              "type": "integer",
+              "format": "int32"
+            }
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "success",
+            "content": {}
+          }
+        }
+      }
+    },
+    "/admin/api/accounts/find.xml": {
+      "get": {
+        "summary": "Account Find",
+        "tags": ["Accounts"],
+        "parameters": [
+          {
+            "name": "access_token",
+            "in": "query",
+            "description": "A personal Access Token",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "username",
+            "in": "query",
+            "description": "Username of the account user.",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "email",
+            "in": "query",
+            "description": "Email of the account user.",
+            "schema": {
+              "type": "string"
+            }
+          },
+          {
+            "name": "user_id",
+            "in": "query",
+            "description": "ID of the account user.",
+            "schema": {
+              "type": "ref"
```

Comment:
There are plenty of `"type": "ref"` entries, this is not valid, all ocurrences need to be fixed.

Reply:
yes agreed I have fixed one so if anything wrong here can be fixed and then rest can be fixed